src/hooks/useCollections.ts

```diff
@@ -181,16 +182,31 @@ export const useCollections = () => {
 
           const unsub = await api.tx.nfts
             .setCollectionMetadata(collectionId, metadataCid)
-            .signAndSend(activeAccount.address, { signer: activeWallet.signer }, ({ status }) => {
+            .signAndSend(activeAccount.address, { signer: activeWallet.signer }, ({ events, status }) => {
               if (status.isReady) {
                 setStatus({ type: ModalStatusTypes.IN_PROGRESS, message: StatusMessages.METADATA_UPDATING });
               }
 
               if (status.isFinalized) {
-                setStatus({ type: ModalStatusTypes.COMPLETE, message: StatusMessages.METADATA_UPDATED });
                 unsub();
 
-                setAction(() => () => navigate(routes.myAssets.collections.index));
+                events.some(({ event: { method } }) => {
+                  if (method === 'ExtrinsicSuccess') {
```

Comment:
can we extract that `'ExtrinsicSuccess'` from the pjs?

Reply:
even the code inside of pjs uses event.method === 'ExtrinsicSuccess', so no